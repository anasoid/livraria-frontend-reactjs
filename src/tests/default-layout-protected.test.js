import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router';
import { Provider } from "react-redux";
import configureStore from 'redux-mock-store';
import thunk from 'redux-thunk';
import DefaultLayoutProtected from '../livraria-router/default-layout-protected';


import initialState from './initial-state';
const middlewares = [thunk];
const mockStore = configureStore(middlewares);
const store = mockStore(initialState);

describe('test DefaultLayoutProtected component', () => {

  it('should renders without crashing', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <MemoryRouter>
          <DefaultLayoutProtected/>
        </MemoryRouter>
      </Provider>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

});