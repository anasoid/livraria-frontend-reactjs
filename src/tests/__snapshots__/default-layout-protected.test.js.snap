// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test DefaultLayoutProtected component should renders without crashing 1`] = `null`;
